Nautono/RTOS_framework_v1/src/sail_i2c.h

```diff
@@ -14,6 +14,12 @@
 typedef enum I2C_DeviceIDs {
```

Comment:
Are all of these devices using a single I2C bus?

Reply:
Yes

Reply:
I2c_compass can be removed from here but I need to remove the sail_compass file for that and make sure nothing else breaks when I do it.